README.md

```diff
@@ -1 +1,15 @@
-# igniteui-angular-i18n
\ No newline at end of file
+# igniteui-angular-i18n
+
+[![Build Status](https://travis-ci.org/IgniteUI/igniteui-angular-i18n.svg?branch=master)](https://travis-ci.org/IgniteUI/igniteui-angular-i18n)
+
+## Purpose
+
+This respository stores the Japanese and Korean resource strings for the Ignite UI for Angular `i18n`. The English resource strings are stored in the [Ignite UI for Angular](https://github.com/IgniteUI/igniteui-angular) repository. It also contains the interfaces for each localizable component's `resourceStrings` property. They can be used as a schema for creating custom resource strings.
+
+## File Usage
+
+An npm package will be produced and it will be referenced in the Ignite UI for Angular product.
```

Comment:
This also not correct - the npm package would be referenced in the clients project alongside with the igniteui-angular.